firefox-ios/firefox-ios-tests/Tests/ClientTests/URLActivityItemProviderTests.swift

```diff
@@ -0,0 +1,40 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UniformTypeIdentifiers
+import XCTest
+@testable import Client
+
+// TODO: FXIOS-10816 Flesh out these unit tests
+final class URLActivityItemProviderTests: XCTestCase {
+    let testWebURL = URL(string: "https://mozilla.org")!
+
+    override func setUp() {
+        super.setUp()
+        DependencyHelperMock().bootstrapDependencies()
+        LegacyFeatureFlagsManager.shared.initializeDeveloperFeatures(with: MockProfile())
```

Comment:
I don't know if the feature flag could change and then invalidate the test ? Is this test dependent on the feature flag value ?

Reply:
Thanks @FilippoZazzeroni, that's a good call. I was actually writing some tests with a mocked Nimbus setup last evening. I'm going to put this into a separate PR for FXIOS-10872. Having a couple issues so I was going to reach out to Roux and I didn't want to hold up this work. 🙂

Reply:
Ah ok no problem @ih-codes that sounds great to me. Feel free to tag me in the upcoming PRs 😃

Reply:
Ok will do @FilippoZazzeroni! 😃

Reply:
Ha! I just figured out why my tests weren't passing, will have a PR up for this shortly and I'll tag you! 😄

Reply:
Ah ok perfect @ih-codes i'm having a look to it